airflow/providers/openlineage/plugins/openlineage.py

```diff
@@ -25,7 +25,7 @@
 
 def _is_disabled() -> bool:
     return (
-        conf.getboolean("openlineage", "disabled")
+        conf.getboolean("openlineage", "disabled", fallback=False)
```

Comment:
@mobuchowski - I had to add explicit fallback here as our "import" check checks the case where the providers are imported directly - and Provider's Manager might not yet have been initialized with provider configurations. So all top-level imports of configuration have to have explicit fallback, despite it being also specified in the provider.yaml

Reply:
Our CI detected it BTW.

Reply:
OK - makes sense. Top level means here "before initialization of plugins", right?

Reply:
yeah

Reply:
Had to also add it for others.